webpack.config.js

```diff
@@ -61,8 +61,23 @@ module.exports = {
     child_process: 'empty',
     net: 'empty'
   },
-  externals: {
-    react: 'react',
-    "react-dom": 'react-dom'
-  }
+  externals: [
+    function(context, request, callback) {
+      if (request === "react" || request === "react-dom") {
+        return callback(null, request);
+      }
+      else if (request.startsWith("azure-devops-ui/Core")) {

```

Comment:
"azure-devops-ui" --> should we make this as const

Reply:
will fix in next PR